react-app/src/api/stakingAPI.ts

```diff
@@ -271,6 +274,89 @@ export const useStakingAPI = (): IStakingAPI => {
     [query.staking]
   );
 
+  const getValidator = useCallback(
+    async (address: string) => {
+      const validatorResponse = await query.staking.validator(address);
+      const validator = validatorResponse.validator;
+
+      if (!validator) {
+        throw Error(`validator with address ${address} not found`);
+      }
+
+      const consensusPubkey = validator.consensusPubkey;
+      if (!consensusPubkey) {
+        throw new Error("Validator consensus public key not found");
+      }
+      const consensusPubAddr = pubKeyToBech32(
+        consensusPubkey,
+        Config.chainInfo.bech32Config.bech32PrefixConsAddr
+      );
+
+      const commissionRaw = validator.commission;
+      const commission = {
+        commissionRates: {
+          // Default cosmos decimal places is 18
+          rate: new BigNumber(
+            commissionRaw?.commissionRates?.rate ?? 0
+          ).shiftedBy(-18),
+          maxRate: new BigNumber(
+            commissionRaw?.commissionRates?.maxRate ?? 0
+          ).shiftedBy(-18),
+          maxChangeRate: new BigNumber(
+            commissionRaw?.commissionRates?.maxChangeRate ?? 0
+          ).shiftedBy(-18),
+        },
+        updateTime: commissionRaw?.updateTime
+          ? new Date(commissionRaw.updateTime.seconds.toNumber())
+          : null,
+      };
+
+      const tokens = {
+        amount: new BigNumber(validator.tokens),
+        denom: CoinMinimalDenom,
+      };
+
+      const unbondingHeight = new BigNumber(
+        validator.unbondingHeight.toString()
+      );
```

Comment:
If this is in `number` in the first place we don't need to convert it to `BigNumber`?